_config.yml

```diff
@@ -9,9 +9,8 @@ exclude: [
   'vendor'
 ]
 
-github:
-  url: https://github.com/betson/irs-efile-viewer/tree/master
-  repository_url: https://github.com/newjersey/taxation-mef-viewer
-
+repository: newjersey/taxation-mef-viewer
 bucket_url: https://jachan-test-cors-public.s3.amazonaws.com
```

Comment:
Curious about this part, how it plays into the rest of our hosting strategy

Reply:
It's a good question! I've removed the S3 bucket in a recent commit: https://github.com/newjersey/taxation-mef-viewer/commit/d5b63efa7f98de17fe6c6eb4dc98052717f43c92

Previously, we hosted the file in the S3 bucket because when we were running this locally, we needed file hosting for our demo file. I couldn't place it in any of the existing buckets because we needed special configuration to add Allow-CORS headers. I had been getting flagged by OIT for making this resource public, which is why I switched off the S3 hosting for this file. 

It was always meant to be temporary until we got the site hosted.

Reply:
Got it, thanks for the rundown 🙏🏼